content/hi/docs/reference/glossary/workload.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Workload
+id: Workloads
+date: 2019-02-13
+full_link: /docs/concepts/workloads/pods/pod-lifecycle/
+short_description: >
+   वर्कलोड कुबेरनेट्स पर चलने वाला एक एप्लिकेशन है।
+
+aka: 
+tags:
+- fundamental
+---
+   वर्कलोड कुबेरनेट्स पर चलने वाला एक एप्लिकेशन है।
+
+<!--more--> 
+
+विभिन्न मुख्य वस्तुएं जो विभिन्न प्रकार या कार्यभार के भागों का प्रतिनिधित्व करती हैं
```

Comment:
I think this way the sentence is easier to understand. WDYT, thoughts @surbhiahuja?
```suggestion
वर्कलोड का विभिन्न प्रकार या कुछ खंड का प्रतिनिधित्व करने वाली विभिन्न कोर ऑब्जेक्ट में
```

Reply:
I still don't think the two sentences together actually make sense.

Reply:
Please could we rewrite this @surbhiahuja ?